### DISCUSSION TITLE
在使用过程中，发现给实体导入模型之后，添加事件，事件会出现偏移

### DISCUSSION THREAD

**Discussion by Wd13869167203** [zouyaoji/vue-cesium#340] [Q&A]
在使用过程中，发现给实体导入模型之后，添加事件，事件会出现偏移

**Comment by zouyaoji** [comment]
请提供复现的例子

**Reply by Wd13869167203** [reply]
![image](https://user-images.githubusercontent.com/31448471/217172658-f017531a-d1cb-4610-89fc-979760851a0a.png)
当父级元素有    transform: scale(1.5);就出现事件偏移了

**Reply by zouyaoji** [reply]
你说的是鼠标事件吧，
你用了transform改了html的元素啊，这个操作有点不太常规哦，估计导致cesium的接口scene.pickPosition出问题了, 我估计用原生Cesium估计也有问题，我解决不了。
